provision/acc_provision/apic_provision.py

```diff
@@ -3325,7 +3284,7 @@ def contract(self, name, subjects=[]):
             children.append(subj)
         return aci_obj("vzBrCP", [('name', name), ('_children', children)])
 
-    def cloudfoundry_tn(self):
+    def cloudfoundry_tn(self, flavor):
```

Comment:
flavor is not getting handled inside the cloudfoundry_tn. It should be defaulted to None